Buildings/ThermalZones/EnergyPlus/UsersGuide.mo

```diff
@@ -33,23 +33,20 @@ The <code>.epw</code> file will be used by the EnergyPlus envelope model, and th
 file will be used by the Modelica model, and must be specified by the parameter <code>weaName</code>
 in the instance <code>building</code>.
 </li>
+</ol>
+The following coupling objects can then be integrated in the model that contains the instance 
```

Comment:
I found peculiar that the instantiation of `ThermalZone` was a non-optional bullet, whereas any other coupling objects was optional. Isn't it technically possible to run Spawn without any `ThermalZone`? This is what I have been doing in the `ZoneSurface` comparison exercise.

I switched this section so that all the coupling objects are presented on the same level (although the ThermalZone is first because it is likely the most useful), without suggesting that some are optional and some are not.